libraries/reconstruction_cuda/cuda_gpu_geo_transformer.cpp

```diff
@@ -597,4 +363,13 @@ void GeoTransformer<T>::checkRestrictions(int splineDegree,
                 "To init function, pass N control points.");
 }
 
+template<typename T>
+std::unique_ptr<float[]> GeoTransformer<T>::copy_out_d_in(size_t size) const {
+        auto copy_in = std::unique_ptr<float[]>(new float[size]);
```

Comment:
can we do it without this extra allocation, e.g. preallocate this in the init() function??

Reply:
Then it should be protected